WebRoot/public/login.jsp

```diff
@@ -731,7 +727,16 @@ function forgotPassword() {
 	window.location.href = url;
 }
 
-function diableEnable(txt) {
+var modal = document.getElementById('ZLoginWhatsThis');
+window.addEventListener('mouseup', function(event) { 
```

Comment:
I think this logic should be tackled with css pseudo classes.

Reply:
have removed this function